setup.cfg

```
@@ -69,6 +69,8 @@ astropy_header = true
 doctest_plus = enabled
 text_file_format = rst
 addopts = --doctest-rst
+filterwarnings =
+    ignore:Can\'t import specreduce_data package:
```

If you want to prevent unhandled warnings from creeping back in...

```suggestion
filterwarnings =
    error
    ignore:Can\'t import specreduce_data package:
```

---

when i add the `error` bit, i get tonnes of failures due to `numpy.ndarray size changed` that are coming from upstream. happens both locally via `pytest` and via `tox` which installs everything fresh via `pip`. added another `ignore` entry to filter that out since it appears to be harmless.

---

You could ignore the numpy warnings like this

https://github.com/astropy/astropy/blob/f96d2aee1b8c29f23f0bdd8a487580f509ca1b6f/setup.cfg#L134-L135

---

yep, this is what i ended up doing.